presto-client/src/types/client.types.ts

```diff
@@ -37,7 +37,7 @@ export interface PrestoResponse {
   error: PrestoError
   id: string
   stats: PrestoStats
-  nextUri: string
+  nextUri: string | undefined
```

Comment:
The client specifically [checks for undefined](https://github.com/prestodb/presto-js-client/pull/29/files#diff-a3fad29ac0e4e9a2e60e9eae9ef04689dbefda9f432dae93d14984be2e04a1d4R319), so updated this + [one additional check](https://github.com/prestodb/presto-js-client/pull/29/files#diff-a3fad29ac0e4e9a2e60e9eae9ef04689dbefda9f432dae93d14984be2e04a1d4R278) for type safety